f8a_report/helpers/report_helper.py

```diff
@@ -162,7 +162,7 @@ def retrieve_stack_analyses_content(self, start_date, end_date):
 
         # Query to fetch Stack Analysis manifests data from start_date to end_date
         query = sql.SQL('SELECT {} FROM {} WHERE {} BETWEEN \'%s\' AND \'%s\'').format(
-            sql.Identifier('requestJson'), sql.Identifier('stack_analyses_request'),
+            sql.Identifier('dep_snapshot'), sql.Identifier('stack_analyses_request'),
```

Comment:
Please make sure that this column in not referenced by name in the result anywhere in the code.